dwave/cloud/computation.py

```diff
@@ -220,13 +258,49 @@ def wait_multiple(futures, min_done=None, timeout=None):
 
     @staticmethod
     def as_completed(fs, timeout=None):
-        """Emulate `concurrent.futures.as_completed()` behavior.
+        """Yield Futures objects as they complete.
+
+        Returns an iterator over the specified list of :class:`Future` objects that
+        yields those objects as they complete. Completion occurs
+        when the submitted job is finished or cancelled.
+
+        Emulates the behavior of the `concurrent.futures.as_completed()` function.
 
-        Returns an iterator over the list of out `Future` instances given by
-        `fs` that yields futures as they complete (finished or were cancelled).
+        Args:
+            fs (list): List of :class:`Future` objects to iterate over.
+            timeout (float, optional, default=None): Maximum number of seconds to await completion.
+                If None, awaits indefinitely.
+
+        Returns:
+            Generator (:class:`Future` objects):
+                Listed :class:`Future` objects as they complete.
+
+        Raises:
+            `concurrent.futures.TimeoutError` is raised if per-future timeout is exceeded.
+
+        Examples:
+            This example creates a solver using the local system's default D-Wave Cloud Client
+            configuration file, submits a simple QUBO problem to a remote D-Wave resource 3 times
+            for differing numers of samples, and yields timing information for each job
+            as it completes.
+
+            >>> import dwave.cloud as dc
+            >>> client = dc.Client.from_config()
+            >>> solver = client.get_solver()
+            >>> u, v = next(iter(solver.edges))
+            >>> Q = {(u, u): -1, (u, v): 0, (v, u): 2, (v, v): -1}
+            >>> computation = [solver.sample_qubo(Q, num_reads=1000),
+            ...                solver.sample_qubo(Q, num_reads=50),
+            ...                solver.sample_qubo(Q, num_reads=10)]   # doctest: +SKIP
+            >>> for tasks in dc.computation.Future.as_completed(computation, timeout=10)
+            ...     print(tasks.timing)   # doctest: +SKIP
+            ...
+            {u'total_real_time': 17318, ... u'qpu_readout_time_per_sample': 123}
+            {u'total_real_time': 10816, ... u'qpu_readout_time_per_sample': 123}
+            {u'total_real_time': 26285, ... u'qpu_readout_time_per_sample': 123}
+            >>> # Snipped above response for brevity
+            >>> client.close()
 
```

Comment:
We can also add example for the task from `wait_multiple` above (get the result of the first computation finished):
```
from dwave.cloud.computation import Future
...
first = next(Future.as_completed(computations))
print(first.timing)
```

Reply:
Added